### PR TITLE
Replaced `ChartUtils.Math` in favour of an extension on `FloatingPoint`

### PR DESCRIPTION
**Pull request by jjatie**
Increases readability, and in many cases removes a set of parentheses to ensure order of operations. Also can be applied to any `FloatingPoint` type, not just `Double` and `CGFloat`

### CODE REVIEW

**Comment by jjatie** [comment]
@liuxuan30 @petester42 This is a quick and easy one.

File: Source/Charts/Utils/ChartUtils.swift [hunk 1]
```diff
@@ -16,9 +16,26 @@ import CoreGraphics
...
+
+    var rad2deg: Self {
+        return self * 180 / .pi
+    }
+
+    /// - returns: An angle between 0.0 < 360.0 (not less than zero, less than 360)
+    /// NOTE: Value must be in degrees
+    var normalizedAngle: Self {
+        let angle = truncatingRemainder(dividingBy: 360)
+        return (sign == .minus) ? angle + 360 : angle
```
Comment (liuxuan30) [inline Source/Charts/Utils/ChartUtils.swift]: the order is flipped. I need to refresh my mind first to see if they are equivalent. Or you already did that?

Comment (jjatie) [reply Source/Charts/Utils/ChartUtils.swift]: They’re equivalent.

Comment (liuxuan30) [reply Source/Charts/Utils/ChartUtils.swift]: made some stupid equations :P

File: Source/Charts/Utils/ChartUtils.swift [hunk 2]
```diff
@@ -16,9 +16,26 @@ import CoreGraphics
     import UIKit
 #endif
 
+extension FloatingPoint {
+    var deg2rad: Self {
```
Comment (liuxuan30) [inline Source/Charts/Utils/ChartUtils.swift]: Generally it's fine. But do you think we should follow the Camel-Case naming? like deg2Rad

Comment (jjatie) [reply Source/Charts/Utils/ChartUtils.swift]: Up to you. I tried `deg2Rad` and `deg2RAD` and they both look silly to me. I also considered `inRadians` and `inDegrees`, but that leaves out grads

Comment (liuxuan30) [reply Source/Charts/Utils/ChartUtils.swift]: OK. How about we keep old `FDEG2RAD` to keep consitent?

Comment (jjatie) [reply Source/Charts/Utils/ChartUtils.swift]: I'll drop the F since it's implied as it's an extension on `FloatingPoint`

**Comment by liuxuan30** [comment]
This is the one of the change I like most, indeed make it more swift and simple

**Pull request merged**